typed-racket-test/fail/issue-169-1.rkt

```diff
@@ -1,5 +1,5 @@
 #;
-(exn-pred #rx"expected: \\(Listof Nothing\\).*given: \\(Listof Pos\\*\\)")
+(exn-pred #rx"expected: \\Null.*given: \\(Listof Pos\\*\\)")
```

Comment:
Right now (6.7) the type of that expression is `(Pairof Nothing (Listof Nothing))`.

In this PR, the type of that expression is simply `Nothing`.